src/main/java/ch/uzh/ifi/hase/soprafs21/controller/PathsApi.java

```diff
@@ -43,21 +42,21 @@
      *         or User unauthenticated (status code 401)
      *         or Resource not found (status code 404)
      */
-    @ApiOperation(value = "Return all paths", nickname = "pathsGet", notes = "", response = ParkDto.class, responseContainer = "List", tags={ "Paths", })
```

Comment:
This file will not be required, as Path is already taken care in another PR

Reply:
This is a generated code